### PR TITLE
UI - Changes to show the embedded view of Asklo component

### PR DESCRIPTION
**Pull request by zellenka**
styles for the embedded chat window.

### CODE REVIEW

File: packages/klevu-ui/src/components/klevu-button/klevu-button.css [hunk 1]
```diff
@@ -12,6 +12,9 @@
   --padding: var(--klevu-button-padding, calc(var(--klevu-size-m) - (var(--klevu-body-s-size) / 2)));
   --font-size: var(--klevu-button-font-size, var(--klevu-body-s-size));
   --text-color: var(--klevu-button-text-color, var(--klevu-color-neutral-1));
+  --icon-size: var(--klevu-icon-size, 1.5em);
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-button/klevu-button.css:15]: rename to --klevu-button-icon-size, same for the other 3

File: packages/klevu-ui/src/components/klevu-button/klevu-button.css [hunk 2]
```diff
@@ -12,6 +12,9 @@
...
+  --icon-color: var(--klevu-icon-color, var(--text-color));
+  --icon-rotation: var(--klevu-icon-rotation, 0deg);
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-button/klevu-button.css:17]: why do we need this, the user can provide a rotated icon?

File: packages/klevu-ui/src/components/klevu-chat-bubble/klevu-chat-bubble.css [hunk 3]
```diff
@@ -11,6 +11,8 @@
   --background-remote: var(--klevu-chat-bubble-background-remote, var(--klevu-color-primary));
   --text-color: var(--klevu-chat-bubble-text-color, inherit);
   --text-color-remote: var(--klevu-chat-bubble-text-color-remote, var(--klevu-color-neutral-1));
+  --border-radius-general: var(--klevu-chat-bubble-border-radius-general, var(--klevu-border-radius-m));
+  --border-radius-corner: var(--klevu-chat-bubble-border-radius-general, 0);
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-chat-bubble/klevu-chat-bubble.css:15]: do we need 2? the user can provide 4 diff values if they want diff radius for each corner?

File: packages/klevu-ui/src/components/klevu-chat-layout/klevu-chat-layout.css [hunk 4]
```diff
@@ -6,10 +6,14 @@
   display: flex !important;
   flex-direction: column;
   max-height: 100vh;
+  --chat-height: var(--klevu-chat-height, 200px);
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-chat-layout/klevu-chat-layout.css:9]: change name of variable to --klevu-chat-layout-height, same for next one

File: packages/klevu-ui/src/components/klevu-chat-layout/klevu-chat-layout.css [hunk 5]
```diff
@@ -6,10 +6,14 @@
...
+  --chat-max-height: var(--klevu-chat-max-height, 100vh);
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-chat-layout/klevu-chat-layout.css:10]: I think we should default to --chat-height.
100vh is the requirement for product-query, we should pass that value from the product-query component

File: packages/klevu-ui/src/components/klevu-chat-layout/klevu-chat-layout.tsx [hunk 6]
```diff
@@ -69,7 +69,9 @@ export class KlevuChatLayout {
   async scrollMainToBottom(behavior: "smooth" | "instant" = "smooth") {
     const instance = await this.#scrollElement?.getInstance()
     if (instance?.customInstance) {
-      instance.customInstance.update(true)
+      setTimeout(() => {
+        instance?.customInstance?.update(true)
+      }, 20)
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-chat-layout/klevu-chat-layout.tsx:74]: why 20?

File: packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.css [hunk 7]
```diff
@@ -4,8 +4,35 @@
    */
   display: block;
   text-align: left;
+  --cornflower-blue: #428ACA;
   --popup-height: var(--klevu-product-query-popup-height, 500px);
   --popup-width: var(--klevu-product-query-popup-width, 520px);
+  --embedded-height: var(--klevu-product-query-embedded-height, 100%);
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.css:10]: change variable names to --klevu-product-query-popup-height, same for others

File: packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.css [hunk 8]
```diff
@@ -4,8 +4,35 @@
...
+  --embedded-width: var(--klevu-product-query-embedded-width, 100%);
+  --embedded-header-background: var(--klevu-product-query-embedded-header-background, var(--cornflower-blue));
+  --embedded-border-color: var(--klevu-product-query-embedded-border-color, var(--cornflower-blue));
+  --embedded-border-radius: var(--klevu-product-query-embedded-border-radius, var(--klevu-border-radius-s));
+  --end-spacer-display: var(--custom-end-spacer-display, block);
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.css:15]: why is this variable name not following pattern?

File: packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.css [hunk 9]
```diff
@@ -157,3 +196,115 @@ klevu-popup {
...
+  padding-bottom: 0;
+  gap: var(--klevu-spacing-05)
+}
+
+.embedded klevu-chat-messages .end-spacer {
+  display: none;
+}
+.embedded klevu-chat-layout {
+  height: calc(var(--embedded-height) - 15px);
+  max-height: calc(var(--embedded-height) - 15px);
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.css:250]: do we need this 15px calc, is there any other way?

File: packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.css [hunk 10]
```diff
@@ -157,3 +196,115 @@ klevu-popup {
...
+.embedded div[part="product-query-popup-header"] .fineprint {
+  display: none;
+}
+
+.embedded .questions-container {
+  gap: 18px;
+  padding: var(--klevu-spacing-05);
+}
+
+/* .embedded .sendmessage {
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.css:264]: remove commented code if not used

File: packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.tsx [hunk 11]
```diff
@@ -456,7 +476,7 @@ export class KlevuProductQueryPopup {
                   <klevu-button
                     originElement={this.originElement}
                     exportparts={partsExports("klevu-button")}
-                    icon="send"
+                    icon="chevron_right"
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.tsx:479]: why did we change this? do we need same in both views?

File: packages/klevu-ui/src/components/klevu-product-query/klevu-product-query.css [hunk 12]
```diff
@@ -1,3 +1,7 @@
 :host {
   display: inline-block;
+  width: var(--klevu-product-query-popup-width, 100%);
+  /* --klevu-product-query-popup-width: 700px;
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-product-query/klevu-product-query.css:4]: remove commented code

File: packages/klevu-ui/src/components/klevu-button/klevu-button.css (hunk 1 above)
Comment (zellenka) [reply packages/klevu-ui/src/components/klevu-button/klevu-button.css]: name changed

**Review by nikhil-narayana-klevu — APPROVED** [review]

**Pull request merged**